Mission: Clarify arducopter differences

hamishwillee opened the pull request.

hamishwillee commented on en/services/mission.md:
```diff
@@ -386,21 +384,9 @@ The following behaviour is not defined by the specification (but is still of int
-
+#### Geofence & Rally Point Plans
 
+ArduCopter supports Geofence and Rally points using this protocol for MAVLink 2 connections.
```
@peterbarker, do all Ardu_variants support geofence and rally points via the protocol - Don was only sure of copter

hamishwillee commented on en/services/mission.md:
```diff
@@ -359,22 +359,20 @@ Mission upload, download, clearing missions, and monitoring progress are support
   An upload that fails (or is canceled) part-way through will not match the pre-update state.
   Instead it may be a mix of the original and new mission.
 - Even if upload is successful, the vehicle mission may not match the version on the uploading system (and if the mission is then downloaded it will differ from the original).
-  - If you try and upload more items than ArduPilot can store the system will "accept" the items (i.e. not report a failure) but will just overwrite each new item to the same (highest) slot in the mission list.
```
Note, @peterbarker, you said this is not true so deleted. Can you check the rest of this and confirm if it is accurate? Feel free to push a pr on the docs for this section.

peterbarker commented:
On Mon, 27 Apr 2020, Hamish Willee wrote:

> @peterbarker, do all Ardu_variants support geofence and rally points via the protocol - Don was only sure of copter

Copter, Rover and Sub support all variants.

Plane supports rally points and missions, but NOT geofences.  That latter 
is in development.

Let's ignore Tracker :-)

hamishwillee commented:
Thanks @peterbarker. I updated what is supported via protocol.

I also added the word "anecdotal" to reflect the different information provided by you and whoever it was gave me those notes (JamesP and Michael). I hope this is sufficient to indicate that the information is not canonical.

If you are able at some point to go over that section and create a PR to clarify this, or remove inaccuracy that would be great. From my perspective I have info from two trustworthy sources, and no time to go digging and check.
